sdk/pinocchio/src/sysvars/slot_hashes/test_std.rs

```diff
@@ -0,0 +1,113 @@
+//! Tests that rely on the `std` feature (host-only helpers, alloc, etc.).
```

Comment:
It looks like all test are run with `std`, so this comment is not accurate.
Does it make sense to move tests from this module into other modules?